### ISSUE TITLE
Training on custom data does not balance out 

### ISSUE THREAD

**Issue by osamamehdi** [POSTECH-CVLab/PyTorch-StudioGAN#163]
Hello, 

thanks for this repo and all of the implementations. I have been trying to get the Implementation to work on some custom data, however, the loss curves always indicate that something is going wrong. 
The data I am training on is 256 x 256 and when training on it the discriminator keeps going to values very close to 0 and eventually either goes to zero or comes back for a few training steps and then repeats the same procedure. 

Do you have any tips or suggestions on which parameters to change so the training runs more stable with a custom dataset?

Thanks

**Comment by MINGUKKANG** [comment]
Some useful tips for GAN training:

1) apply weight decay (1e-5)
2) use differentiable augmentations
3) remove attention layers
4) use smaller batch size for training.

Luck luck:)
